### PR TITLE
Add CommittedAt to the scm options

### PR DESCRIPTION
**Pull request by BenMusch**
Codeclimate test reporter requires it: https://github.com/codeclimate/test-reporter/blob/f3d2357c7e0684b0105ac77fbb3ba3b2c82c1d70/man/cc-test-reporter-env.1.md

### CODE REVIEW

**Comment by bfosberry** [comment]
👍 If that means anything :P

**Comment by BenMusch** [comment]
@bfosberry

**Comment by GeorgeMac** [comment]
we miss you @bfosberry

File: scm.go [hunk 1]
```diff
@@ -44,6 +44,7 @@ type GitCheckoutOptions struct {
 	Branch            string
 	CommitID          string
 	CommitMessage     string
+	CommittedAt       int64
```
Comment (markphelps) [inline scm.go]: could prob use a uint64 here so it'll continue to work in future millennia ;)

**Pull request merged**